### PR TITLE
Update ReadRobotsDataAdapterTest.java

### PR DESCRIPTION
**Pull request by Nachopm98**
balblabla

### CODE REVIEW

File: VolkswagenCleaningRobots_test_FAILED_TODO/src/test/java/com/volkswagen/adapter/ReadRobotsDataAdapterTest.java [hunk 1]
```diff
@@ -14,6 +14,7 @@ class ReadRobotsDataAdapterTest {
     void creates_a_read_robots_data_adapter() {
         ReadRobotsDataAdapter readRobotsDataAdapter = new ReadRobotsDataAdapter();
         assertThat(readRobotsDataAdapter.readDataFrom(DEFAULT_FILE_PATH)).isNotEmpty();
+        // TODO: HI
```
Comment (bocdagla) [inline VolkswagenCleaningRobots_test_FAILED_TODO/src/test/java/com/volkswagen/adapter/ReadRobotsDataAdapterTest.java:17]: WHY ADD A COMMENTASJDKANDKJSNDKLAS

**Review by bocdagla — CHANGES_REQUESTED** [review]
WHY ARE YOU LIKE THIS!

File: VolkswagenCleaningRobots_test_FAILED_TODO/src/test/java/com/volkswagen/adapter/ReadRobotsDataAdapterTest.java (hunk 1 above)
Comment (jacekust) [reply VolkswagenCleaningRobots_test_FAILED_TODO/src/test/java/com/volkswagen/adapter/ReadRobotsDataAdapterTest.java:17]: Can you make whole class a oneliner please

**Review by bocdagla — APPROVED** [review]
Nice 🗡️

**Pull request merged**